cmd/gvproxy/main.go

```diff
@@ -101,6 +105,18 @@ func main() {
 			exitWithError(errors.Errorf("%q already exists", uri.Path))
 		}
 	}
```

Comment:
all other sockets compare to `""` to determine empty. why length?

Reply:
The blocks just above `if len(vfkitSocket) > 0 { ... }` are `if len(qemuSocket) > 0 { ... }` and `if len(bessSocket) > 0 { ... }`.

There are `""` comparisons after this block but for a different purpose (error checks for conflicting options) which I forgot to add to this PR, I'll do this.